app/helpers/application_helper.rb

```diff
@@ -1,13 +1,13 @@
 module ApplicationHelper
```

Comment:
ignore these changes - there aren't any changes but line spacing, I was considering adding stuff but deleted it.